spatialprofilingtoolbox/cggnn/scripts/run.py

```diff
@@ -164,8 +164,8 @@ def save_importances(_args):
 
 if __name__ == "__main__":
     args = parse_arguments()
-    study_data: Dict[str, Dict] = FeatureMatrixExtractor.extract(
-        args.spt_db_config_location)[args.study]
+    extractor = FeatureMatrixExtractor(database_config_file=args.spt_db_config_location)
+    study_data: Dict[str, Dict] = extractor.extract(study=args.study)
```

Comment:
No need to use the capital D `Dict` in Python 3.11.

Reply:
Yup, fixed.